lib/local-links-manager/import/links_importer.rb

```diff
@@ -26,27 +26,59 @@ def initialize(csv_downloader = CsvDownloader.new(CSV_URL, header_conversions: F
         @missing_record_count = 0
         @missing_id_count = 0
         @invalid_record_count = 0
+        @deleted_record_count = 0
+        @links_in_csv = {}
```

Comment:
Minor: You could use a [`Set`](http://ruby-doc.org/stdlib-2.3.1/libdoc/set/rdoc/Set.html) here rather than using a hash like one.  I think you'll get the same performance characteristics, but the intent might be clearer - while reading the code I was momentarily confused about what the values in the hash were, when in fact we don't care about them at all, only the keys.